packages/cli/src/commands/codegen.ts

```diff
@@ -54,6 +57,7 @@ export const codegen: Command = {
         })}`
       )
       .option(`-b, --bindgen <URI>`, `${intlMsg.commands_codegen_options_b()}`)
+      .option(`-e, --embed`, `${intlMsg.commands_codegen_options_e()}`)
```

Comment:
Currently the CLI isn't able to detect if the abi-bindgen wrap supports this feature. Not sure how to best handle this, maybe we should instead bump the feature version of the abi-bindgen interface, and the CLI could see if the implementation supports this feature?

This doesn't matter too much, but just a thought.

Reply:
My assumption was that the `embed-bindgen` wraps are quick and easy to write, so we would support all of the languages soon

Reply:
Kris updated the implementation within the `wrap-abi-bindgen` repo to use the embeds from the `config` property on the wrap. This makes it an "optional" feature and there is no harm done if a wrap does not support it, the data will simply not be used by the bindgen wrap.